src/_gettsim/transfers/kinderzuschl/kinderzuschl_eink.py

```diff
@@ -21,15 +22,18 @@ def kinderzuschl_bruttoeink_eltern_m(
     ----------
     arbeitsl_geld_2_bruttoeink_m
         See :func:`arbeitsl_geld_2_bruttoeink_m`.
-    eltern
-        See :func:`eltern`.
+    kindergeld_anspruch
+        See :func:`kindergeld_anspruch`.
+    erwachsen
+        See basic input variable :ref:`erwachsen <erwachsen>`.
```

Comment:
This is a function. 

I think we should be able to get rid of the `kind` input variable now? Based on hh and elternteil pointers, we should be fine, no?

I find kind/erwachsen super confusing because it is so ambiguous. In the legal sense it is clear, age 18 is the border. But this somehow uses the SGB II definition (docstring says *Dependent child living with parents*), but we try to set that up in a more sensible way now?

Reply:
To some extent `kind` can actually be replaced by `kindergeld_anspruch`, I think. 

But we really need to check if different definitions apply in other areas of law (SGB II, whatever). But I think it often says "Kinder im Sinne des Bundeskindergeldgesetzes".

Reply:
Yes definitely. But I think we should do this in a new PR.

For ALG2 Regelbedarf, "Erwachsen" seems to refer to anyone over 18 (independent of Kindergeld Anspruch). So there are definitely differences.